### DISCUSSION TITLE
Add a quick Mute/Unmute and PTT fixed on mobile

### DISCUSSION THREAD

**Discussion by L4ky** [edumeet/edumeet#868] [Ideas]
Hi, 
i think it would be awesome if there are 2 buttons with fixed position (so always visible) with Mute/Unmute function and PTT, in the bottom right corner ( where usually there is nothing since most of the smartphones have the camera on top ).

This feature would be needed only on mobile devices.
It's a bit unhandy to tap on the video then tap on mute/unmute. 
Often i disable my webcam by mistake.

**Comment by Astagor** [comment]
I do not agree. It was like this in the previous version and the buttons were on videos when there are more participants. I think it is good how it is now. Of course it could be configurable were the buttons are, but I think it is not something critical.

**Comment by havfo** [comment]
I guess this is fixed with the "separate media buttons" option?
